index.html

```diff
@@ -0,0 +1,47 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" href="styles.css" />
+    <title>Andres -- Martin</title>
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id="main" class="main">
+      <h1 id="title">Nombre Blog</h1>
+      <p>mensaje titulo</p>
+      <figure id="img-div">
+          <div>
+              <img id="image" src="img/computador.jpg"  />
+              <figcaption id="img-caption">
+                  "Infografia de la imagen"
+              </figcaption>
+          </div>
+      </figure>
+      <section id ="tribute-info">
```

Comment:
kindly the section's id has a nondescriptive name. You might consider changing it to something more representative of its content.